### DISCUSSION TITLE
M2M how to get the actual values

### DISCUSSION THREAD

**Discussion by BrunoUY** [directus/directus#14586] [Archive]
Hi,

I have one collection that has a M2M relationship.

When I run a HTTP get or using the .readByOne method on the collection I get the value as for example
ex of query:
GET http://localhost:8055/items/assets_watchlists/1
Response:
`
"assets_id": [
            2,
            3,
            4,
            5,
            6,
            72
        ]
`

those are the ID's of the M2M table.

Currently I'm doing a second request to the M2M table to get the actual Id's from the "assets_id"
GET http://localhost:8055/items/assets_watchlists_assets/2
Response
`
"data": {
        "assets_asset_id": 11656,
        "assets_watchlists_asset_watchlist_id": 1,
        "id": 2
    }
`

**Question:**
There is any way to get the actual ID's ( in this case assets_asset_id 11656 )that live inside the M2M table, and avoid the second step to get those values ?

thank you, Bruno.

**Comment by rijkvanzanten** [comment]
https://docs.directus.io/reference/introduction/#relational-data

https://docs.directus.io/reference/query/#fields

**Comment by BrunoUY** [comment]
Thank you @rijkvanzanten
